docs/concepts/knowledge.mdx

```diff
@@ -57,6 +57,24 @@ crew = Crew(
 result = crew.kickoff(inputs={"question": "What city does John live in and how old is he?"})
 ```
 
+## Appending Knowledge Sources To Individual Agents
+Sometimes you may want to append knowledge sources to an individual agent. You can do this by setting the `knowledge` parameter in the `Agent` class.
+
+```python
+agent = Agent(
+    ...
+    knowledge={
```

Comment:
Why do you have multiple metadata in Knowledge?

Also, what happens if someone doesn't provide metadata? What happens?

Reply:
An error is raised. Chromadb complains too